client/src/utilities/linkeditemsrepository.cpp

```diff
@@ -169,6 +169,7 @@ void LinkedItemsRepository::removeNote(const QString &id)
             const int idx = std::distance(notes.constBegin(), it);
             kDebug() << "Removing note at" << idx;
             notes.remove(idx);
+            emit opportunityModified(oldAccountId);
```

Comment:
oldAccountId is an account, not an opportunity.

Reply:
Done